dataactcore/interfaces/function_bag.py

```diff
@@ -296,7 +296,7 @@ def mark_job_status(job_id, status_name):
 
     # if status is changed to finished for the first time, check dependencies
     # and add to the job queue as necessary
-    if old_status != 'finished' and status_name == 'finished':
+    if old_status != 'finished' and status_name == 'finished' and not skip_check:
```

Comment:
Just making sure, this doesn't prevent cross validation from happening because one of its dependencies won't be "finished"